.bashrc

```diff
@@ -80,8 +82,38 @@ if [ -x /usr/bin/dircolors ]; then
     alias egrep='egrep --color=auto'
 fi
 
+# Initiate virtualenv
 WORKON_HOME="/home/vagrant/.virtualenvs"
 source /usr/local/bin/virtualenvwrapper.sh > /dev/null 2>&1
 cd /vagrant
 
-# the workon command to be added during provisioning 
+# Ask for git configuration data from user based on input
+chmod 755 /home/vagrant/gitconfig
+export PATH=$PATH:/home/vagrant/
+echo 'export PATH=$PATH:/home/vagrant/' >> ~/.profile
```

Comment:
I think that it is correct and I supply a screenshot for you to determine. Let me run some research and see if that is really correct. So you are suggesting to put 
`export 'PATH=$PATH:/home/vagrant'`

<img width="1010" alt="1__developer__vagrant_" src="https://cloud.githubusercontent.com/assets/10653912/15462209/61ce0560-20c1-11e6-98a8-4182489804a2.png">